zk/tests/common.py

```diff
@@ -0,0 +1,75 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import os
+from datadog_checks.utils.common import get_docker_hostname
+
+CHECK_NAME = 'zk'
+HOST = get_docker_hostname()
+PORT = 12181
+HERE = os.path.dirname(os.path.abspath(__file__))
+URL = "http://{}:{}".format(HOST, PORT)
+INSTANCE = {
+    'host': HOST,
+    'port': PORT,
+    'expected_mode': "standalone",
+    'tags': ["mytag"]
+}
+
+WRONG_EXPECTED_MODE = {
```

Comment:
can we make these fixtures for consistency and to make the code more pytest-idiomatic?